core/src/main/python/wlsdeploy/tool/util/alias_helper.py

```diff
@@ -522,6 +522,20 @@ def get_model_topology_top_level_folder_names(self):
         """
         return self.__aliases.get_model_topology_top_level_folder_names()
 
+    def get_model_resources_top_level_folder_names(self):
```

Comment:
Why is it necessary to add these methods to the helper?  Isn't it sufficient to access the underlying alias method directly from the unit test?  I don't see the reason to expose this as a "public" API.

Reply:
I'm confused. 

First, all of the other public methods on the aIiases.py are exposed on the alias_helper.py, so I assumed that that was the design intention of the helper. The way RP explained it, the xxx_helper was there so the caller wouldn't have to have a try/except around every call they make. 

Second, I didn't do this code change because a unit test was bombing out. As I stated in the comment for the issue, I did the code change because the usage_printer.py is bombing out due to the missing helper methods. I think you might be surmising that since usage_printer.py isn't one of the files I needed to change but a unit test is, that this change isn't really needed. The unit test file was changed because the existing unit tests weren't written in a way to catch the error, in the first place.